### ISSUE TITLE
CvOverflowMenu does not have aria-label props

### ISSUE THREAD

**Issue by lbenicio-ibm** [carbon-design-system/carbon-components-vue#1387]
## Detailed description

The component Cv Overflow Menu, which create an overflow menu with a button as trigger, does not support appropriated aria-label props on that button.

> Describe in detail the issue you're having.

I am having difficulties in creating accessibility features for screen readers.

> Is this a feature request (new component, new icon), a bug, or a general issue?

Bug

> Is this issue related to a specific component?

Cv Overflow Menu

> What did you expect to happen? What happened instead? What would you like to see changed? 

I expected to pass prop "aria-label" to Cv Overflow Menu components, and it properly adds this prop to its button.

> What browser are you working in?

Safari

> What version of the Carbon Design System are you using?

10.46.0

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?

No

## Steps to reproduce the issue

1. Create a Cv-Overflow-Menu component
2. Add "aria-label" prop
3. Inspect element on generated html in browser
4. generated button does not show proper aria-label

## Additional information

- Screenshots or code
- Notes
